modules/planning/tasks/deciders/lane_change_decider/lane_change_decider.cc

```diff
@@ -200,13 +208,16 @@ void LaneChangeDecider::PrioritizeChangeLane(
 
 void LaneChangeDecider::RemoveChangeLane(
     std::list<ReferenceLineInfo>* reference_line_info) const {
+  return;
+  /* disable rest to avoid core dump */
+  ADEBUG << "removed change lane";
   auto iter = reference_line_info->begin();
   while (iter != reference_line_info->end()) {
     if (iter->IsChangeLanePath()) {
-      iter = reference_line_info->erase(iter);
-    } else {
-      ++iter;
+      reference_line_info->erase(iter);
```

Comment:
Please don't change here as there might be more than one lane_change reference_line in the list to remove.